### ISSUE TITLE
JWT shim does not exist for Bun

### ISSUE THREAD

**Issue by ottomated** [muxinc/mux-node-sdk#368]
The package.json contains this:
```json
"bun": {
  "types": "./_shims/auto/*.d.ts",
  "require": "./_shims/auto/*-bun.js",
  "default": "./_shims/auto/*-bun.mjs"		
},
```

But `_shims/auto/jwt-bun.js` does not exist, as well as `_shims/auto/types-bun.js`. This causes the entire library to break when using Bun.

This is entirely blocking our upgrade to version 8 of the SDK.

**Comment by RobertCraigie** [comment]
Thanks for the report, this should be fixed in the next release with #369.

**Comment by RobertCraigie** [comment]
This fix has been released in `v8.3.1`.
